apps/passport-server/src/services/poapService.ts

```diff
@@ -235,7 +358,7 @@ export class PoapService {
 
           const newPoapLink = await claimNewPoapUrl(
```

Comment:
in the `lock.acquire` above (not changed in this PR) - to be precise, i wonder if it's worth locking on the concatenation of the `ticketId` and `poapEvent`, rather than just on the `ticketId`.

Reply:
It is unlikely but possible that this race condition would occur in practice, and given it's so easy I think it's worth doing.